src/patternfly/components/Menu/examples/Menu.md

```diff
@@ -0,0 +1,837 @@
+        {{/menu-item}}
+      {{/menu-list-item}}
+    {{/menu-list}}
+  {{/menu-body}}
+{{/menu}}
+```
+
+### With-actions
+```hbs
+{{#> menu}}
+  {{#> menu-body}}
+    {{#> menu-group}}
+      {{#> menu-group-title}}
+        Actions
+      {{/menu-group-title}}
+      {{#> menu-list}}
+        {{#> menu-list-item}}
+          {{#> menu-item menu-item--modifier="pf-m-selected"}}
+            {{#> menu-item-main}}
+              {{#> menu-item-text}}
+                Item 1 
+              {{/menu-item-text}}
+              {{> menu-item-select-icon}}
+            {{/menu-item-main}}
+            {{#> menu-item-description}}
+              This is a description
+            {{/menu-item-description}}
+          {{/menu-item}}
+          {{#> menu-item-action}}
+            <i class="fas fa-fw fa-ellipsis-v" aria-hidden="true"></i>
+          {{/menu-item-action}}
+        {{/menu-list-item}}
+        {{#> menu-list-item}}
+          {{#> menu-item menu-item--modifier="pf-m-select"}}
+            {{#> menu-item-main}}
+              {{#> menu-item-text}}
+                Item 2
+              {{/menu-item-text}}
+              {{> menu-item-select-icon}}
+            {{/menu-item-main}}
+          {{/menu-item}}
+          {{#> menu-item-action}}
+            <i class="fas fa-fw fa-bell" aria-hidden="true"></i>
+          {{/menu-item-action}}
+        {{/menu-list-item}}
+        {{#> menu-list-item}}
+          {{#> menu-item menu-item--modifier="pf-m-selected"}}
+            {{#> menu-item-main}}
+              {{#> menu-item-text}}
+                Item 3
+              {{/menu-item-text}}
+              {{> menu-item-select-icon}}
+            {{/menu-item-main}}
+            {{#> menu-item-description}}
+              This is a description
+            {{/menu-item-description}}
+          {{/menu-item}}
+          {{#> menu-item-action}}
+            <i class="fas fa-fw fa-clipboard" aria-hidden="true"></i>
+          {{/menu-item-action}}
+        {{/menu-list-item}}
+        {{#> menu-list-item}}
+          {{#> menu-item menu-item--modifier="pf-m-selected"}}
+            {{#> menu-item-main}}
+              {{#> menu-item-text}}
+                Item 4
+              {{/menu-item-text}}
+              {{> menu-item-select-icon}}
+            {{/menu-item-main}}
+            {{#> menu-item-description}}
+              This is a description
+            {{/menu-item-description}}
+          {{/menu-item}}
+          {{#> menu-item-action}}
+            <i class="fas fa-fw fa-bars" aria-hidden="true"></i>
+          {{/menu-item-action}}
+        {{/menu-list-item}}
+      {{/menu-list}}
+    {{/menu-group}}
+  {{/menu-body}}
+{{/menu}}
+```
+
+### With-favorites
+```hbs
+{{#> menu}}
+  {{#> menu-body}}
+    {{#> menu-group}}
+      {{#> menu-group-title}}
+        Favorites
+      {{/menu-group-title}}
+      {{#> menu-list}}
+        {{#> menu-list-item}}
+          {{#> menu-item menu-item--type="a"}}
```

Comment:
these are missing an `href`. You could use `--IsLink` or something to generate the element and attributes. You also need to add `type="button"` to the actions which could be done as an `--IsAction` or `{{# unless --IsLink}}`